api/v1/relay/relay.proto

```diff
@@ -0,0 +1,152 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import public "flow/flow.proto";
+
+package relay;
+
+service HubbleRelay {
```

Comment:
I think this could simply be `Relay` instead of `HubbleRelay` (there is not hubble prefix for either of the peer or observer services).